rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/widget/OneScanCovidRDTBarcodeFactory.java

```diff
@@ -25,6 +26,7 @@ protected void launchBarcodeScanner(Activity activity, MaterialEditText editText
         InputMethodManager inputManager = (InputMethodManager) activity.getSystemService(Context.INPUT_METHOD_SERVICE);
         inputManager.hideSoftInputFromWindow(editText.getWindowToken(), HIDE_NOT_ALWAYS);
         Intent intent = new Intent(activity, OneScanActivity.class);
+        intent.putExtra(Constants.Config.ENABLE_BATCH_SCAN, widgetArgs.getJsonObject().optBoolean(Constants.Config.ENABLE_BATCH_SCAN, false));
```

Comment:
I think it's enough to just have `.optBoolean(Constants.Config.ENABLE_BATCH_SCAN)` since it defaults to false.